templates/index.html

```diff
@@ -56,12 +60,16 @@
         {{ coffee.description }}
       </div>
       <div class="extra">
-        <div class="ui label">{{ coffee.notes }}</div>
+      {% for note in coffee.notes.replace('&', ',').split(',') %}
```

Comment:
i wonder if we can do this (the replace and split) in the backend instead. I dont like doing this stuff on the front end